Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3702,15 +3702,27 @@ def get_modified_remote_data_command(client: Client, params: dict[str, str],
     if not last_update_time:
         last_update_time = remote_args.last_update
     last_update = get_time_parameter(last_update_time, epoch_format=True)
-    filter_ = f'id <= {highest_fetched_id} AND ((status!=closed AND last_persisted_time > {last_update}) OR (status=closed AND close_time > {last_update}))'  # noqa: E501
-    print_debug_msg(f'Filter to get modified offenses is: {filter_}')
+    assert isinstance(last_update, int)
```

Comment:
Is there always a value in `last_update`?
If there isn't (e.g. First Run,) then this `assert` will fail?

Reply:
yes, we make sure that the context is valid in every mirroring call